packages/manager/modules/anthos/src/dashboard/general-information/upgrade-pack/upgrade-pack.html

```diff
@@ -0,0 +1,54 @@
+<div>
+    <oui-header
+        data-heading="{{:: ('anthos_tenant_dashboard_general_information_upgrade_pack_title' | translate) }}"
+        data-description="{{:: ('anthos_tenant_dashboard_general_information_upgrade_pack_description' | translate) }}"
+    >
+        <oui-back-button data-on-click="$ctrl.goBack()"></oui-back-button>
+    </oui-header>
+    <ovh-manager-product-offers
+        data-workflow-type=":: $ctrl.workflowType"
+        data-workflow-options=":: $ctrl.workflowOptions"
+        data-pricing-type=":: $ctrl.pricingType"
+        data-on-error=":: $ctrl.onError(error)"
+        data-on-success=":: $ctrl.onSuccess(result)"
+        data-send-current-state="$ctrl.onSendCurrentState(state)"
+    >
+        <oui-step-form
```

Comment:
```oui-step-form``` must be child of ```oui-stepper``` component.
Why you don't use the parent tag ?

Reply:
Because the `ovh-manager-product-offers` component uses `oui-stepper` as its root element and transcludes everything as children of it

Reply:
Ah ok, got it.
Thanks.